payment-processor/src/main/java/Invoice.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Invoice {
```

Comment:
Adotar um único idioma para o código é essencial para se ter consistência e legibilidade. O que você acha de alterar os nomes das classes para português ou as variáveis e métodos para inglês para ter uma padronização no idioma utilizado no código?

Reply:
I'm gonna change the class names to Portuguese :+1: